src/main/java/org/broadinstitute/hellbender/engine/dataflow/DataflowCommandLineProgram.java

```diff
@@ -71,11 +70,11 @@ public static String getRunnerTypeName(String runnerSimpleName) {
     protected String stagingLocation;
 
     @Argument(doc = "path to the client secrets file for google cloud authentication. Ignored if apiKey is set.",
-            shortName = "secret", fullName = "client_secret", optional=true)
-    protected File clientSecret = new File("client_secret.json");
+            shortName = "secret", fullName = "client_secret", optional=true, mutex={"apiKey"})
+    protected File clientSecret;
 
     @Argument(doc = "API Key for Google Cloud (unnecessary if a client secrets file is provided).",
-            shortName = "apiKey", fullName = "apiKey", optional=true)
+            shortName = "apiKey", fullName = "apiKey", optional=true, mutex={"client_secret"})
     protected String apiKey = null;
```

Comment:
Can we create a ticket to make dataflow authentication easier (eg., automatically populate the API key from the appropriate environment variable)?